pyxem/signals/electron_diffraction2d.py

```diff
@@ -184,11 +184,19 @@ def set_diffraction_calibration(self, calibration, center=None):
         calibration : float
             Diffraction pattern calibration in reciprocal Angstroms per pixel.
         center : tuple
-            Position of the direct beam center, in pixels. If None the center of
-            the data array is assumed to be the center of the pattern.
+            Position of the direct beam center, in calibrated units. If None the center of
+            the data array is assumed to be the center of the pattern unless `center_pixels` is given.
+        center_pixels : tuple
+            Position of the direct beam center, in pixels. Will only be used if `center` is not given.
```

Comment:
```suggestion
        pixels : bool
            If True, the provided center is in pixel coordinates. Default is False which means
            the provided center is in calibrated coordinates.
```

Reply:
Sorry @CSSFrancis, I suddenly got tied up with parental leave - but I'll take a look and commit your suggestion and the related change in the unit tests this week!